doc/userguide/rules/integer-keywords.rst

```diff
@@ -0,0 +1,74 @@
+.. _rules-integer-keywords:
+
+Integer Keywords
+================
+
+Many keywords will match on an integer value on the network traffic.
+These are unsigned integers that can be 8, 16, 32 or 64 bits.
+
+Simple example::
+
+    bsize:integer value;
+
+The integer value can be written as base-10 like ``100`` or as 
+an hexadecimal value like ``0x64``.
+
+The most direct exemple is to match for equality, but there are
+different modes.
+
+Comparison modes
+----------------
+
+Integers can be matched for
+* Equality
+* Inequality
+* Greater than
+* Less than
+* Range
+* Negated range
+* Bitmask
+* Negated Bitmask
+
+Comparisons are strict by default.
+That means a range between 1 and 4 will match 2 and 3, but neither 1 nor 4.
```

Comment:
This connects to @jlucovsky question here, right? https://github.com/OISF/suricata/pull/10208#discussion_r1460916496

I think it's good to make this into a note or something that is highlighted a bit more.

Reply:
>This connects to @jlucovsky question here, right? https://github.com/OISF/suricata/pull/10208#discussion_r1460916496

Perfectly right.

How do I make a note ?

Reply:
Just added a suggestion that shows how to do it :)